spikeinterface/toolkit/postprocessing/tests/test_correlograms.py

```diff
@@ -19,5 +25,21 @@ def test_compute_correlograms():
     correlograms, bins = compute_correlograms(sorting2)
 
 
+def test_compute_autocorrelogram_from_spiketrain():
```

Comment:
you should add a pytest.skip if not have numba here

Reply:
Done.